templates/cisco_ios_show_mac-address-table.textfsm

```diff
@@ -42,4 +42,5 @@ TYPE4
   ^Displaying\s+entries
   ^Vlan\s+Mac Address\s+Type\s+Ports
   ^\s*$$
+  ^Total\s+Mac\s+Addresses\s+for\s+this\s+criterion\:\s+\d+
```

Comment:
```suggestion
  ^Total\s+Mac\s+Addresses
```
I think we can capture less and still be confident is the line we think it is, while allowing syntax to be slightly different across platforms and OS versions